Frontend/Gmfctn/src/app/modules/+auth/components/sign-in/sign-in.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, EventEmitter, OnDestroy, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { Subscription } from 'rxjs';
+import { finalize, take } from 'rxjs/operators';
+import { AuthenticateService } from '../../services/authenticate.service';
+
+@Component({
+  selector: 'app-sign-in',
+  templateUrl: './sign-in.component.html',
+  styleUrls: ['./sign-in.component.scss'],
+})
+export class SignInComponent implements OnInit, OnDestroy {
+  errorMessage = '';
+  loading = false;
+  subscription$ = new Subscription();
```

Comment:
You don't need this $ sign at the end of the variable name. It is used as a convention to mark variables as a stream, but a subscription Object is not a stream.

Reply:
Fixed